API/controllers/sessionController.js

```diff
@@ -1,79 +1,77 @@
-import jwt from 'jsonwebtoken';
-import { users, mentors, sessions } from '../models/data';
-
+import { users, sessions } from '../models/data';
+import Responses from '../helpers/response';
 
 export default class Sessions {
   // user should request a session to mentor
   static requestSession(req, res) {
     const session = {
       sessionId: sessions.length + 1,
       mentorId: req.body.mentorId,
-      menteeId: req.userInfo.user.id,
+      menteeId: req.userInfo.id,
       questions: req.body.questions,
-      menteeEmail: req.userInfo.user.email,
+      menteeEmail: req.userInfo.email,
       status: 'pending',
     };
     sessions.push(session);
-    res.status(200).json({
-      message: 'session requested successfully!',
-      data: session,
+    return Responses.success(res, 200, 'session requested successfully!', session);
+    // res.status(200).json({
+    //   message: 'session requested successfully!',
+    //   data: session,
 
-    });
+    // });
   }
 
-  // mentor accepting menotorship request session
-  static acceptSession(req, res) {
-    const session = sessions.find((s) => s.sessionId === parseInt(req.params.sessionId, 10));
-
+  // eslint-disable-next-line consistent-return
+  static declineSession(req, res) {
+    // eslint-disable-next-line no-shadow
```

Comment:
@habaumugisha1 , kindly remove all suspended eslint flags. the aim of the `airbnb` style guide is to enforce the methodology. hence you should not ignore the eslint flag with a comment